wrap/README.md

```diff
@@ -8,9 +8,40 @@ It was designed to be more general than just wrapping GTSAM. For notes on creati
 
 1. This library uses `pybind11`, which is included as a subdirectory in GTSAM.
 2. The `interface_parser.py` in this library uses `pyparsing` to parse the interface file `gtsam.h`. Please install it first in your current Python environment before attempting the build.
-    ```
-    python3 -m pip install pyparsing
-    ```
+
+```
+python3 -m pip install pyparsing
+```
+
+## Getting Started
+
+Clone this repository to your local machine and perform the standard CMake install:
+
+```sh
+mkdir build && cd build
+cmake ..
+make install # use sudo if needed
+```
+
+Using `wrap` in your project is straightforward from here. In you `CMakeLists.txt` file, you just need to add the following:
+
+```cmake
+include(PybindWrap)
```

Comment:
Again, if this goes with every GTSAM then probably we will need to change the name to be `GtwrapPybindWrap` or similar.

Reply:
I agree

Reply:
This should actually be fine since it installs under the `gtwrap` directory for cmake.